Content.Server/Traits/Assorted/UnrevivableSystem.cs

```diff
@@ -5,11 +5,11 @@ namespace Content.Server.Traits.Assorted;
 
 public sealed class UnrevivableSystem : EntitySystem
 {
-    public override void Initialize()
+   public override void Initialize()
     {
-        base.Initialize();
+       base.Initialize();
```

Comment:
It appears you removed one space from the indentation here too
```suggestion
        base.Initialize();
```